crates/test-utils/src/network.rs

```diff
@@ -257,3 +260,22 @@ pub async fn start_fullnode_from_config(
         ws_url,
     })
 }
+
+pub async fn wait_for_nodes_transition_to_epoch<'a>(
```

Comment:
there may be an opportunity to remove some duplicated effort here - `trigger_reconfiguration` (in reconfiguration_tests.rs) does almost the same thing

Reply:
nevermind i see you already did that